tests/test_gbs.py

```diff
@@ -0,0 +1,608 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the GBS device.
+"""
+import pennylane as qml
+import pytest
+from pennylane import numpy as np
+from pennylane.operation import Probability
+from pennylane.qnodes.base import ParameterDependency
+from pennylane.wires import Wires
+from strawberryfields.ops import GraphEmbed, MeasureFock
+from strawberryfields.program import Program
+
+from pennylane_sf import StrawberryFieldsGBS
+from pennylane_sf.ops import ParamGraphEmbed
+from pennylane_sf.simulator import StrawberryFieldsSimulator
+
+target_cov = np.array(
+    [
+        [
+            2.2071e00,
+            1.2071e00,
+            1.2071e00,
+            1.2071e00,
+            -2.6021e-18,
+            -1.3878e-16,
+            6.9323e-17,
+            -7.8505e-17,
```

Comment:
Oh, black :(

If you want this to be formatted more like a matrix, feel free to use `#fmt: off` and `fmt: on` to turn black on and off around this array.